test/string_view-test/string_view-test.cpp

```diff
@@ -0,0 +1,349 @@
+/*
+* string_view-test.cpp
+* Ray Chen, Sean Murthy 
+* (c) 2020 sigcpp https://sigcpp.github.io. See LICENSE.MD
+*
+* Attribution and copyright notice must be retained.
+* - Attribution may be augmented to include additional authors
+* - Copyright notice cannot be altered
+* Attribution and copyright info may be relocated but they must be conspicuous.
+*
+* Test the string_view of the C++ standard library and 
+* provide a test benchmark for subsequent implementation of string_view.
+* - see C++17 [string.view]
+*	https://timsong-cpp.github.io/cppwp/n4659/string.view
```

Comment:
Simplify prologue comment

```suggestion
* Unit tests for string_view
* - see C++17 [string.view]
* - https://timsong-cpp.github.io/cppwp/n4659/string.view
```